pymeasure/experiment/parameters.py

```diff
@@ -46,9 +46,12 @@ class Parameter:
 
     def __init__(self, name, default=None, ui_class=None, group_by=None, group_condition=True):
         self.name = name
-        self._value = default
+        self._value = None
+        if default is not None:
+            self._value = self.convert(default)
```

Comment:
You can consider using the setter here; that way you ensure setting the default value gets the same treatment as does normal value setting (for now not so much difference, but you never know):
```suggestion
            self.value = default
```